src/action-creators/alert.ts

```diff
@@ -18,22 +19,26 @@ let clearAlertTimeoutId: number | null = null // eslint-disable-line fp/no-let
  * @param clearDelay Timeout after which alert will be cleared.
  */
 const alert =
-  (value: string | FunctionComponent | null, { alertType, showCloseLink, clearDelay }: Options = {}): Thunk =>
+  (
+    value: string | FunctionComponent | null,
+    { alertType, showCloseLink, clearDelay, isInline = false }: Options = {},
+  ): Thunk =>
   (dispatch, getState) => {
     const { alert } = getState()
 
     if (clearTimeout) {
       // if clearAlertTimeoutId !== null, it means that previous alert hasn't been cleared yet. In this case cancel previous timeout and start new.
       clearAlertTimeoutId && clearTimeout(clearAlertTimeoutId)
-      clearAlertTimeoutId = setTimeout(() => {
```

Comment:
Never delete code without providing the following:

- Code to replace the intended functionality
- OR a justification of why the intended functionality is no longer relevant/needed

In this case, we need `clearAlertTimeoutId` to clear the previous timeout, otherwise previous alerts with a longer clear delay could still be dispatched. A new alert should override all previous and pending alerts.

Reply:
Got you! Thanks